types/luxon/src/settings.d.ts

```diff
@@ -65,6 +66,15 @@ export class Settings {
      * Reset Luxon's global caches. Should only be necessary in testing scenarios.
      */
     static resetCaches(): void;
+
+    static get defaultWeekSettings(): WeekSettings | null;
+
+    /**
+     * Allows overriding the default locale week settings, i.e. the start of the week, the weekend and
+     * how many days are required in the first week of a year.
+     * Does not affect existing instances.
+     */
+    static set defaultWeekSettings(weekSettings: WeekSettings | null);
```

Comment:
These could just be the property
```suggestion
    /**
     * Allows overriding the default locale week settings, i.e. the start of the week, the weekend and
     * how many days are required in the first week of a year.
     * Does not affect existing instances.
     */
    static defaultWeekSettings: WeekSettings | null;
```

Reply:
Landed in [ee29728](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/68088/commits/ee297285c6d37b3165c13ccaf37f8c9c37b9cc26)

Thank you for the review! 🙌